src/js/classes/Table.js

```diff
@@ -338,9 +321,8 @@ class BodyRow {
 
     row.className = "";
     this.cells.forEach((cell, i) => {
-      cell.setElementClass(
-        i === sorted ? `${cell.className} sorted` : cell.className
-      );
+      const classNames = `${cell.className} ${i === sorted ? "sorted" : ""} ${this.isBold ? "bold-cell" : ""}`;
+      cell.setElementClass(classNames);
```

Comment:
Let's use `classList.add` here instead:
```
cell.classList.add(cell.className);
if (i === sorted) cell.classList.add("sorted");
if (this.isBold) cell.classList.add("bold-cell");
```

Reply:
Had to start with an initial `setElementClass` call because cell.className could contain multiple whitespace-separated tokens, which `classList.add` doesn't like.

Reply:
Yeah, that's why I said to split it up. (See example code in my comment above.)

Reply:
I meant that the value in `cell.className` has multiple class names in the string (ignoring "sorted" or "bold-cell") so the `cell.classList.add(cell.className)` doesn't work. I could do a `.split(" ")` but I went with:
```

Reply:
Ah, alright